rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -78,12 +78,18 @@ class BakeryController {
     return baseImage
   }
 
-  @ExceptionHandler
+  @ExceptionHandler(BakeOptions.Exception)
   @ResponseStatus(HttpStatus.NOT_FOUND)
   Map handleBakeOptionsException(BakeOptions.Exception e) {
     [error: "bake.options.not.found", status: HttpStatus.NOT_FOUND, messages: ["Bake options not found. " + e.message]]
   }
 
+  @ExceptionHandler(Exception)
```

Comment:
What's the reason for this change?

I don't know enough spring boot to understand the implications without more context.

How does this change what is returned in the case of an exception? Do any changes need to be made to the callers of this entry-point?